products/compute/terraform.yaml

```diff
@@ -317,6 +317,8 @@ overrides: !ruby/object:Provider::ResourceOverrides
     properties:
       id: !ruby/object:Provider::Terraform::PropertyOverride
         name: proxyId
+      sslPolicy: !ruby/object:Provider::Terraform::PropertyOverride
+        exclude: false
```

Comment:
I like the reverse exclude, but you should check with @nelsonjr - we're having a similar argument about whether that's appropriate on a different PR.

Reply:
I like it too. If we find it brings confusion in the future, we can reevaluate and ban the practice. I personally find it more elegant. I added TODO linking to the bug to enable it in C/P/A.

Additionally, for this particular case, a validation happens before the overrides are applied which would cause the ResourceRef check to fail.